src/operations/search/query/r4.js

```diff
@@ -122,7 +122,23 @@ module.exports.buildR4SearchQuery = (resource_name, args) => {
         columns.add('meta.lastUpdated');
     }
 
-    for (const [field, filterObj] of Object.entries(customQueries)) {
+    for (const [resourceType, filterObj] of Object.entries(customScalarQueries)) {
```

Comment:
Is it possible to use the Array.filter() method to filter out the 'resourceType === resource_name' to help reduce nesting?

Reply:
this is unpacking so I'm not sure how we could filter...